core/src/main/resources/lib/form/password/password.css

```diff
@@ -27,9 +27,9 @@
 }
 
 .hidden-password-legend {
-    border: 1px solid #ccc;
+    border: 1px solid var(--input-border);
```

Comment:
For CSS adjuncts the fallback must be done by hand. For example, this case would be 

````
border: 1px solid #ccc;
border: 1px solid var(--input-border, #ccc);
````

This is one of the reasons of why I wanted to kill adjuncts.